docs/reference/docs/reindex.asciidoc

```diff
@@ -1119,6 +1122,30 @@ POST _reindex
 // TEST[s/otherhost:9200/\${host}/]
 // TEST[s/\.\.\.,/"username": "test_admin", "password": "x-pack-test-password",/]
 
+[[ccs-based-reindex-from-remote]]
+==== ccs based reindex from remote
+
+Reindex supports Cross-cluster search (<<modules-cross-cluster-search,{ccs}>>) based reindexing from a remote Elasticsearch cluster:
+
+[source,console]
+--------------------------------------------------
+POST _reindex
+{
+  "source": {
+    "index": "cluster_one:my-index-000001"
+  },
+  "dest": {
+    "index": "my-new-index-000001"
+  }
+}
+--------------------------------------------------
+// TEST[setup:my_index_big]
+
+The `index` parameter must be specified with the remote cluster name and the index name, separated by 
+a colon (e.g., cluster_one:my-index-000001).
+Additionally, you must configure the remote cluster in advance and retrieve the remote cluster name.
```

Comment:
I don't follow what "retrieve the remote cluster name" means here.
Perhaps, it should all be rephrased more simply such as: "Note that reindexing requires that the remote cluster be configured on the local cluster."

Reply:
That certainly seems confusing.
I'll rephrase it to a simpler version.

Reply:
[simple note](https://github.com/elastic/elasticsearch/pull/117154/commits/6ae57e05cdb51ec530aab31bcdc1cf7a7a987eca)